charts/onroutebc/values.yaml

```diff
@@ -34,7 +35,7 @@ frontend:
       ACCESS_API_URL: "https://{{.Release.Name}}-vehicles.apps.silver.devops.gov.bc.ca"
       POLICY_URL: "https://{{.Release.Name}}-policy.apps.silver.devops.gov.bc.ca"
       DEPLOY_ENVIRONMENT: "{{.Values.global.vault.zone}}"
-      FRONTEND_PR_NUM: "{{.Values.global.pr_num}}"
+      PR_NUM: "{{.Values.global.env.pr_num}}"
```

Comment:
This change has to be made in Line 47 below.
`"VITE_FRONTEND_PR_NUM":"$FRONTEND_PR_NUM",`

Also, you've to add release number to the frontend.
`"VITE_RELEASE_NUMBER":"$RELEASE_NUMBER"`

Reply:
Will do, cheers!